src/main/java/service/impl/FruitDalyTransactionsHandlerImpl.java

```diff
@@ -0,0 +1,43 @@
+package service.impl;
+
+import db.FruitsStorage;
+import java.util.List;
+import model.FruitTransactionModel;
+import service.FruitDalyTransactionsHandler;
+import strategy.DalyOperationStrategy;
+
+//this class calculate balance of fruits according to the strategy
+// - received List<FruitTransactionModel> and DalyOperationStrategy
+// - return an instance of FruitsStorage
+public class FruitDalyTransactionsHandlerImpl implements FruitDalyTransactionsHandler {
```

Comment:
You should have interface OperationHandler and then such implementation:
- BalanceOperationHandlerImpl
- PurchaseOperationHandlerImpl
- ReturnOperationHandlerImpl
- SupplyOperationHandlerImpl

Reply:
Okay) But... In my opinion, this is too complicated a structure for this case... I did it... but it's hard for me to imagine what kind of logic I should put in all these implementations))